### PR TITLE
Fix CI for azure/travis

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Get rid of unnecessary spacing in logging.

### CODE REVIEW

File: ci/set-branch.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    esac
+    shift
+done
+
+echo -e "\e[35m\e[1mREPO   = ${REPO}\e[0m"
+echo -e "\e[35m\e[1mBRANCH = ${BRANCH}\e[0m"
+
+echo -e "\e[35m\e[1mGetting default branch by GitHub API of: ${REPO}\e[0m"
+
+DEFAULT_BRANCH=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/"${REPO}" 2>/dev/null | jq ".default_branch" -r)
```
Comment (ar13pit) [inline ci/set-branch.sh:29]: Why not use the git repo itself to get the default branch? That would free us from using the Github API.

**Review by ar13pit — CHANGES_REQUESTED** [review]

File: ci/set-branch.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+if [[ "$BRANCH" == "$DEFAULT_BRANCH" ]]
+then
+    echo -e "\e[35m\e[1mOn the default branch, clearing BRANCH variable\e[0m"
+    BRANCH=""
+else
+    echo -e "\e[35m\e[1mOn a feature branch, keeping the BRANCH variable: ${BRANCH}\e[0m"
+fi
+
+export BRANCh
```
Comment (ar13pit) [inline ci/set-branch.sh:34]: Nice typo @MatthijsBurgh :laughing: 
```suggestion
export BRANCH
```

**Review by ar13pit — CHANGES_REQUESTED** [review]

File: ci/set-branch.sh (hunk 2 above)
Comment (MatthijsBurgh) [reply ci/set-branch.sh]: Keeping you sharp ;)

**Comment by ar13pit** [comment]
My only question now is why not use the same logic in GitHub Actions?

**Comment by MatthijsBurgh** [comment]
> y not use the same logic i

This would require cloning of the repo in single package repo's, which we don't do now. This would require changing the current configs

**Review by ar13pit — APPROVED** [review]

**Pull request merged**